src/qutip_qip/device/cavityqed.py

```diff
@@ -75,32 +75,31 @@ class DispersiveCavityQED(ModelProcessor):
     num_levels: int, optional
         The number of energy levels in the resonator.
 
-    deltamax: int or list, optional
-        The coefficients of sigma-x for each of the qubits in the system.
-
-    epsmax: int or list, optional
-        The coefficients of sigma-z for each of the qubits in the system.
-
-    w0: int, optional
-        The base frequency of the resonator.
-
-    eps: int or list, optional
-        The epsilon for each of the qubits in the system.
-
-    delta: int or list, optional
-        The epsilon for each of the qubits in the system.
-
-    g: int or list, optional
-        The interaction strength for each of the qubit with the resonator.
-
-    t1: list or float
+    t1: list or float, optional
         Characterize the decoherence of amplitude damping for
         each qubit. A list of size `num_qubits` or a float for all qubits.
 
-    t2: list of float
+    t2: list of float, optional
         Characterize the decoherence of dephasing for
         each qubit. A list of size `num_qubits` or a float for all qubits.
 
+    **params:
+        Keyword argument for hardware parameters, in the unit of GHz.
+        Qubit parameters can either be a float or a list of the length
+        ``num_qubits``.
+
+        - ``deltamax``: the pulse strength of sigma-x control, default ``1.0``
+        - ``epsmax``: the pulse strength of sigma-z control, default ``9.5``
+        - ``eps``: the epsilon for each of qubit, default ``10``
+        - ``delta``: the delta for each of qubit, default ``0.0``
```

Comment:
```suggestion
        - ``eps``: the epsilon for each of qubit, default ``10``
        - ``delta``: the delta for each of qubit, default ``0.0``
```
Can you add something more descriptive here for `eps` and `delta`?